tests/cypress/page-object/sitemap.page.ts

```diff
@@ -0,0 +1,17 @@
+import { BasePage } from './base.page'
+import { editPage } from './edit.page'
+import { workflowDashboard } from './worflow.dashboard.page'
+
+class DigitallHomePage extends BasePage {
```

Comment:
naming "DigitallHomePage" ?

Reply:
removed that file and class, it wasn't used by anything